predicators/pybullet_helpers/robots/__init__.py

```diff
@@ -1,27 +1,35 @@
 """Handles the creation of robots."""
-from typing import Dict
+from typing import Dict, Type
 
 from predicators.pybullet_helpers.geometry import Pose, Pose3D, Quaternion
 from predicators.pybullet_helpers.robots.fetch import FetchPyBulletRobot
+from predicators.pybullet_helpers.robots.panda import PandaPyBulletRobot
 from predicators.pybullet_helpers.robots.single_arm import \
     SingleArmPyBulletRobot
 
 # Note: these are static base poses which suffice for the current environments.
 _ROBOT_TO_BASE_POSE: Dict[str, Pose] = {
     "fetch": Pose(position=(0.75, 0.7441, 0.0)),
+    "panda": Pose(position=(0.8, 0.7441, 0.25)),
+}
+
+_ROBOT_TO_CLS: Dict[str, Type[SingleArmPyBulletRobot]] = {
+    "fetch": FetchPyBulletRobot,
+    "panda": PandaPyBulletRobot,
 }
```

Comment:
We could use the same strategy in env and dynamically determine which robots are available

Reply:
Won't bother for now